Makefile

```diff
@@ -273,6 +273,8 @@ endif
 	docker exec gardener-local$(TARGET_SUFFIX)-control-plane sh -c "sysctl fs.inotify.max_user_instances=8192" # workaround https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
 	cp $(KUBECONFIG) $(REPO_ROOT)/example/provider-local/seed-kind$(TARGET_SUFFIX)/base/kubeconfig
 	kubectl apply -k $(REPO_ROOT)/example/gardener-local/calico --server-side
+	kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.6.1/components.yaml
+	kubectl patch -n kube-system deployment metrics-server --type=json -p '[{"op":"add", "path":"/spec/template/spec/containers/0/args/-", "value":"--kubelet-insecure-tls"}]'
```

Comment:
Instead of applying the manifest first and then patching the deployment, you could also use kustomize like we do for the other components that we deploy to kind.
I.e.:
```
	kubectl apply -k $(REPO_ROOT)/example/gardener-local/metrics-server --server-side
```

And add a `example/gardener-local/metrics-server/kustomization.yaml` and `example/gardener-local/metrics-server/patch-metrics-server.yaml` similar to `example/provider-local/garden/local/patch-controller-deployment.yaml` (merge patch) or `example/provider-local/seed-kind2/local/patch-seed.yaml` (JSON patch).

Reply:
Thank you!